### PR TITLE
[VEN-2925]: add USDC market on zksync

### PR DESCRIPTION
**Pull request by GitGuru7**
## NOTE

Resolves VEN-2925

### CODE REVIEW

**Comment by GitGuru7** [comment]
> Sepolia commands lgtm

https://sepolia.explorer.zksync.io/tx/0x2c6e155d736dd2a8d0a453a697853de95969e35341cb4a91bffb1fdaf85c9233#overview

**Review by chechu — COMMENTED** [review]
Mainnet commands lgtm

File: multisig/simulations/zksyncmainnet/vip-015/index.ts
[the remark this answers is not in the document]
Comment (chechu) [reply multisig/simulations/zksyncmainnet/vip-015/index.ts:1]: Check USDC price: https://github.com/VenusProtocol/vips/pull/421/commits/257e23644832f0aedb91ba532b2499543926b5a4

I think the Generic IL test is not available for zkSync

**Review by kkirka — APPROVED** [review]
Risk parameters, deployments, commands LGTM

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/392?chainId=56

File: multisig/simulations/zksyncmainnet/vip-015/index.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+const { zksyncmainnet } = NETWORK_ADDRESSES;
+
+const COMPTROLLER_CORE = "0xddE4D098D9995B659724ae6d5E3FB9681Ac941B1";
+const PSR = "0xA1193e941BDf34E858f7F276221B4886EfdD040b";
+
+const BLOCKS_PER_YEAR = BigNumber.from("31536000"); // equal to seconds in a year as it is timebased deployment
+
+forking(48211836, async () => {
+  describe("Post-Execution state", () => {
+    let allMarkets: any;
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply multisig/simulations/zksyncmainnet/vip-015/index.ts]: [081e82c](https://github.com/VenusProtocol/vips/pull/421/commits/081e82cf4481bbe3e98caf530f9744136aa7af74)

**Review by kkirka — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**